src/reference/modules/administration/pages/server.adoc

```diff
@@ -1,5 +1,23 @@
 = Server administration
 
+== Configuration repository
+
+The configuration defined on the server is stored in a local git repository, in `/var/rudder/configuration-repository`.
+
+This repository contains the history of the configuration, and in some cases may also manage
+the shared files in `/var/rudder/configuration-repository/shared-files`.
+
+NOTE: The *xml* files in this repository should generally not be edited outside of Rudder
```

Comment:
>*xml* files in this repository

This is misleading since the line just above mention the shared-files dir, which is not the target here.